cdds/cdds/common/plugins/file_info.py

```diff
@@ -119,16 +119,18 @@ def get_date_range(self, nc_files: List[str], frequency: str) -> Tuple[TimePoint
         file_end = TimePointParser().strptime(file_match.group('end_date'),
                                               strptime_format_string=OUTPUT_FILE_DT_STR[frequency]['str'])
 
-        # For subhrPt frequency the seconds can be either the model time step or the radiation time step (1hr)
-        seconds_for_delta = OUTPUT_FILE_DT_STR[frequency]['delta'][1]
-        if seconds_for_delta is None:
+        # for the end date, we want the start of the next day for easier processing.
+        # So if the range is 20100101-20191230, use 20200101 as the end date.
+        if frequency == "subhrPt":
+            # For subhrPt frequency the seconds can be either the model time step or the radiation time step (1hr)
             last_file_end = re.search(self._nc_files_to_archive_regex, nc_files[-1]).group('end_date')
-            # Assuming all time steps are an integer number of minutes
             seconds_for_delta = 60 * (60 - int(last_file_end[10:12]))
+            delta_to_add = Duration(days=OUTPUT_FILE_DT_STR[frequency]['delta'][0], seconds=seconds_for_delta)
```

Comment:
Searching through, this line is the *only* remaining reference to the delta information in the `OUTPUT_FILE_DT_STR` anywhere in CDDS, and for the subhrPt frequency this is zero, so I suggest
```suggestion
            delta_to_add = Duration(days=0, seconds=seconds_for_delta)
```
At which point we can remove the `'delta'` key in OUTPUT_FILE_DT_STR.